_events/2022-0509.markdown

```diff
@@ -0,0 +1,18 @@
+---
+
+eventdate: 2022-05-09T09:00
+tz: UTC -7
+title: OpenSearch Community Meeting - 2022-05-09
+online: true
+signup:
+    url: https://www.meetup.com/OpenSearch/events/285396734/
+    title: Join on Meetup
+
+---
+
+Join us for our weekly online community meeting.
```

Comment:
I might change the 'weekly' to be more clear that this format isn't weekly

Reply:
made a suitable change to address this.